install_config/aggregate_logging.adoc

```diff
@@ -96,14 +85,14 @@ $ oc project logging
 [NOTE]
 ====
 Specifying a non-empty
-link:../admin_guide/managing_projects.html#using-node-selectors[node
+xref:../admin_guide/managing_projects.html#using-node-selectors[node
 selector] on the project is not recommended, as this would restrict
 where Fluentd can be deployed. Instead, specify node selectors for the
-deployer to be applied on your other deployment configurations.
+deployer to be applied to your other deployment configurations.
 ====
 
-. Create a link:../dev_guide/secrets.html[secret] for the deployer. The secret
-can be configured specifically for your instance of the deployer.
+. Create a xref:../dev_guide/secrets.html#[secret] for the deployer. The secret
+can be configured specifically for the deployer.
```

Comment:
I'll admit, i was questioning the second sentence myself, but you reworded into what it is today, so I thought it was relevant. I largely kept with what you suggested, but took out "optional". Does that mean the contents of the secret are optional? I feel that requires a lot of explanation. Unless you mean the secret itself is optional? In this case, we need to change up a lot of stuff I think...